### DISCUSSION TITLE
Ability to filter on any column in a page

### DISCUSSION THREAD

**Discussion by kranskydog** [jhuckaby/Cronicle#670] [Ideas]
For example, Upcoming Events has filters for Categories, Plugins, Servers, whereas Completed only has a filter for Event, even though it also has columns for Categories, Plugins, Servers

Don't know if it can be done, but would be good if the basic pattern was to just make every column heading in a page a filter

**Comment by jhuckaby** [comment]
Done in Cronicle v2, which is a complete redesign of the back-end storage system (it's now more of a "database").  This would be far too difficult to backport to v0, unfortunately.
